int8 way_robot
int8 wait_robot
int8 path_num
int8 wait_num